scripts/stitch.py

```diff
@@ -345,15 +345,15 @@ def create_doc(images_path, ext=None):
     # Loop for all initial chunks
     for chunk in chunk_list:
         doc.chunk = chunk
-        
+
     # Align Photo only if it is not done yet
     if chunk.point_cloud is None:
         AlignPhoto(chunk, Accuracy, Key_Limit, Tie_Limit)
         ReduceError_RU(chunk)
         ReduceError_PA(chunk)        
     # Do the rest when there's tie point
         ReduceError_RE(chunk)
-        StandardWorkflow(doc, chunk, args.out,
+        StandardWorkflow(doc, chunk, str(Path(args.out).resolve()),
```

Comment:
We use `Path().resolve()` to convert the output path (ie `args.out`) to an absolute path if it isn't already an absolute path.
Then, we convert the `Path` to a `str` and pass it as an argument to the StandardWorkflow function.